query-engine/driver-adapters/executor/script/testd.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+node "$(dirname "${BASH_SOURCE[0]}")/../dist/testd.mjs"
```

Comment:
This is just a subject of a rename, the directory structure was really weird when I introduced the benchmarking code, which reused a lot of the connector-test-kit-executor (all except the index.ts file)

To ease the review, you can collapse connector-test-kit-executor, which doesn't exist anymore, and look into the new files in executor.

In particular: `bench.ts` and `recordReplay.ts`